x/incentive/client/cli/tx.go

```diff
@@ -33,7 +38,175 @@ func GetTxCmd() *cobra.Command {
 	cmd.AddCommand(CmdSetWithdrawAddress())
 	cmd.AddCommand(CmdWithdrawValidatorCommission())
 	cmd.AddCommand(CmdWithdrawDelegatorReward())
+	cmd.AddCommand(CmdSubmitUpdateMinCommissionRateProposal())
+	cmd.AddCommand(CmdSubmitUpdateMaxVotingPowerProposal())
+	cmd.AddCommand(CmdSubmitUpdateMinSelfDelegationProposal())
 	// this line is used by starport scaffolding # 1
 
 	return cmd
 }
+
+func CmdSubmitUpdateMinCommissionRateProposal() *cobra.Command {
```

Comment:
I am not sure if this proposal needs to be put on incentive module - since it's just for ante handler.